app/src/main/java/org/mozilla/fenix/tabstray/browser/TabSorter.kt

```diff
@@ -57,6 +58,8 @@ class TabSorter(
         if (shouldReportMetrics) {
             shouldReportMetrics = false
 
+            Metrics.inactiveTabsCount.set(inactiveTabs.size.toLong())
```

Comment:
I know this is not your fault but is it possible for us to go through `GleanMetricsService` when calling `Metrics`?  I think this is the only case that call directly through `Metrics`.